readthedocs/projects/static-src/projects/js/import.js

```diff
@@ -282,6 +282,8 @@ function ProjectImportView(instance, config) {
             .then(function (data) {
                 self.get_organizations();
                 self.get_accounts();
+                // re-sync
+                self.filter_by(self.filter_by());
```

Comment:
I'm not sure if this is the proper way of doing this, but I didn't find anything on the docs. But I tested this and works as expected.

Reply:
Did you test it locally? The line looks fine to me.

Don't we need to re-request the projects also and then apply the filter?

Reply:
> Did you test it locally? The line looks fine to me.

Yes, I tested locally, it works.

> Don't we need to re-request the projects also and then apply the filter?

When the filter is set, a new request is made https://github.com/rtfd/readthedocs.org/pull/4113/files#diff-fd74ba503cde92cdf9a406ce7da32f2eL210.